docs/app/Examples/views/Card/Variations/Colored.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Card, Image } from 'stardust'
+
+const ColoredCards = () => (
+  <div>
+    <Card.Group itemsPerRow={4}>
+      <Card color='red'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='orange'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='yellow'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='olive'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='green'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='teal'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='blue'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='violet'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='purple'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='pink'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='brown'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='grey'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+    </Card.Group>
+  </div>
```

Comment:
I initially thought this too, but the issue I ran into with implementing this is, in placing the image within a prop, you lose control of where the image will be placed within the Card. Based on the semantic examples: http://semantic-ui.com/views/card.html#types

An image can go anywhere within the card, inbetween content or at the top, or the bottom - image does not have a particular position the same way it does within `<Feed.Label />` components.